### ISSUE TITLE
[FEATURE] Single-Connector Caching

### ISSUE THREAD

**Issue by xCykrix** [amethyst-studio/k-value#28]
**Is your feature request related to a problem? Please describe.**
Data consistency for a single-connection application required calls for each request.

**Describe the solution you'd like**
Design a mode that will cache selected requests in memory, invalidating them on a set operation. This will only function consistently if only one application is connected to the database backend storage table or namespace. If the stored data changes, it will only be available until the cached items are manually invalidated or expire due to requests being routed to memory instead of the database and will not be merged correctly when #23 is implemented.

The purpose of this feature is to handle high loads on a single application service that may have a higher number of reading vs writing operations.

Feature requested by @ghostdevv

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by ghostdevv** [comment]
Eyy

**Comment by xCykrix** [comment]
Landed in 222c470 [Milestone 2.0.0 Development] and set for release next publish period.

Breaking changes are pending release with this milestone.

Functional Example:
```js
Adapter.get('id', { cache: true }) // Update-cache value
Adapter.set('id', newValue) // Invalidate-cache value
```

/cc @ghostdevv
